observatorium/overlays/moc/smaug/loki/deployments/observatorium-loki-distributor.yaml

```diff
@@ -0,0 +1,139 @@
+kind: Deployment
```

Comment:
seems like this deployment exists
https://github.com/operate-first/apps/blob/master/observatorium/base/instance/loki/distributor/deployment.yaml